pom.xml

```diff
@@ -74,9 +74,6 @@
             <archive>http://groups.google.com/group/killbilling-dev</archive>
         </mailingList>
     </mailingLists>
-    <prerequisites>
-        <maven>3.0</maven>
-    </prerequisites>
```

Comment:
Any reason for dropping the `prerequisites` ?

Reply:
`prerequisites` as a top-level section is only meaningful for Maven plugins. Instead, we should be using the `maven-enforcer-plugin` if we really wanted to enforce the Maven version.

I looked into it but:

* I don't fully remember why we needed Maven 3.0+ (could be related to some of the plugins)
* Maven 3.0 was released back in 2010, so we should be safe :-)